docs/tutorials/functions.ipynb

```diff
@@ -4,9 +4,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# User Functions\n",
+    "# Functions\n",
     "\n",
-    "User functions allow to add or overwrite functions in the tax and transfer system provided by gettsim. There are several ways to pass functions to the `user_functions` argument of `compute_taxes_and_transfers()`."
+    "Functions allow to add or overwrite functions in the tax and transfer system provided by gettsim. There are several ways to pass functions to the `functions` argument of `compute_taxes_and_transfers()`."
```

Comment:
To keep it short, I wanted it to deal with functions only and I wanted it to explain the module since no one except you has probably take a look at #198.